server/protocols/jmap-draft/src/test/java/org/apache/james/jmap/mailet/filter/JMAPFilteringTest.java

```diff
@@ -1033,6 +1036,65 @@ void actionShouldSupportKeywords(JMAPFilteringTestSystem testSystem) throws Exce
             .containsOnly("abc", "def");
     }
 
+    @Test
```

Comment:
We need more tests, obvously...

Integration tests like JAMES-3944 looks like a good starting place.